tools/kraken.py

```diff
@@ -69,10 +72,10 @@ def pipeline(self, inBam, db, outReport=None, outReads=None, filterThreshold=Non
                            JVMmemory=picard.jvmMemDefault,
                            background=True)
 
-            if numThreads is None:
+            if not numThreads:
```

Comment:
If we're leaving `None` as the sentinel value (instead of 0), then I would prefer the`is None` conditional.

Reply:
true.. I think I was defending against zero, but that was before I worked my way into changing all the argparse defaults more thoroughly..